apiml/src/main/java/org/zowe/apiml/controller/InvalidWebFingerConfigurationException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.controller;
+
+public class InvalidWebFingerConfigurationException extends RuntimeException {
+
+    private final String messageId;
+
+    public InvalidWebFingerConfigurationException(Throwable cuase, String messageId) {
+        super(cuase);
```

Comment:
```suggestion
    public InvalidWebFingerConfigurationException(Throwable cause, String messageId) {
        super(cause);
```

Missing serialUid. Why have we started providing the messageId in the exception? It should be based on the exception type in an ExceptionHandler...